src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Color[] Colors = Color.values();
+    private static final Random Random = new Random();
```

Comment:
java code convention violation.

```suggestion
    private static final Color[] colors = Color.values();
    private static final Random random = new Random();
```